fluent-package/Gemfile

```diff
@@ -37,21 +37,22 @@ end
 
 # plugin gems
 
-gem "elasticsearch", "8.7.0"
+gem "elasticsearch", "8.8.0"
 gem "fluent-plugin-elasticsearch", "5.3.0"
 gem "ruby-kafka", "1.5.0"
 gem "digest-murmurhash", "1.1.1"
 gem "fluent-plugin-kafka", "0.19.0"
-gem "prometheus-client", "2.1.0"
+gem "prometheus-client", "4.1.0"
+# f-p-prometheus_pushgateway doesn't support f-p-prometheus 2.1.0 or later
```

Comment:
Oops, we need to release fluent-plugin-prometheus_pushgateway that support it.

Reply:
https://github.com/fluent/fluent-plugin-prometheus_pushgateway/pull/13

Reply:
resolved.